### DISCUSSION TITLE
Add where the post is hosted

### DISCUSSION THREAD

**Discussion by Rem0ld** [dailydotdev/daily#736] [Feature Requests]
We should see before redirection where it's headed.
Medium is not free and we can read only 2 posts a month.
It would spare time to see before clicking if we can access the post.

**Comment by Rem0ld** [comment]
After looking more precisely, it's already there but might not be clear enough.

**Comment by idoshamun** [comment]
Hi @Rem0ld,

Thanks for raising this concern. We do our best to remove Medium's paywall content.
Not all Medium articles are paywalled so most of them on daily.dev should be free
